python/docs/Makefile

```diff
@@ -55,125 +25,3 @@ html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

Comment:
So this is the only target we actually need? I'm Ok with it if we can't think of any reason to use the other targets. I assume this whole thing was a copy paste.

Reply:
As far as I can tell, `html` and `clean` are the only targets we use.